lib/src/layout_element.dart

```diff
@@ -40,7 +40,10 @@ class TableLayoutElement extends LayoutElement {
               return FractionColumnWidth(width);
             } else {
               final width = double.tryParse(widthStr);
-              return FixedColumnWidth(width);
+              if (width != null) {
```

Comment:
I think that a ternary looks better here. Something like : `return (width != null ? FixedColumnWidth(width) : null);`

Reply:
Yes, it's better. Fixed.